ple/games/donkeykong/Wall.py

```diff
@@ -0,0 +1,19 @@
+__author__ = 'Batchu Vishal'
+from OnBoard import OnBoard
+import pygame
+
+'''
+This class defines all our walls in the game.
+Currently not much is done here, but we can add traps to certain walls such as spiked walls etc to damage the player
+'''
+
+
+class Wall(OnBoard):
+    def __init__(self, raw_image, position, rng):
+        super(Wall, self).__init__(raw_image, position)
```

Comment:
Doesnt `OnBoard` expect an rng value in its `__init__` function? Could you correct this in throughout the code.